docs/source/json-api/index.rst

```diff
@@ -1234,6 +1234,401 @@ HTTP Response
+      "rights": [
+        {
+          "type": "CanActAs",
+          "party": "Carol"
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Alice",
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Bob",
+        },
+        {
+          "type": "ParticipantAdmin"
+        }
+      ]
+    }
+
+Please refer to :ref:`CreateUserRequest <com.daml.ledger.api.v1.admin.user_management_service.CreateUserRequest>` documentation for information about the meaning of the fields.
+
+All fields in the request are required, this means that an empty JSON object is not a valid request to allocate a new user.
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": true,
+      "status": 200
+    }
+
+
+Get Authenticated User Information
+**********************************
+
+This endpoint exposes the Ledger API's :ref:`GetUserRequest <com.daml.ledger.api.v1.admin.user_management_service.GetUserRequest>`.
+
+The user ID will always be filled out with the user specified via the currently used user token.
+
+HTTP Request
+============
+
+- URL: ``/v1/user``
+- Method: ``GET``
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": {
+        "userId": "Carol",
+        "primaryParty": "Carol",
+      },
+      "status": 200
+    }
+
+
+Get Specific User Information
+*****************************
+
+This endpoint exposes the Ledger API's :ref:`GetUserRequest <com.daml.ledger.api.v1.admin.user_management_service.GetUserRequest>`.
+
+HTTP Request
+============
+
+- URL: ``/v1/user``
+- Method: ``POST``
+- Content-Type: ``application/json``
+- Content:
+
+.. code-block:: json
+
+    {
+      "userId": "Carol"
+    }
+
+
+Please refer to :ref:`GetUserRequest <com.daml.ledger.api.v1.admin.user_management_service.GetUserRequest>` documentation for information about the meaning of the fields.
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": {
+        "userId": "Carol",
+        "primaryParty": "Carol",
+      },
+      "status": 200
+    }
+
+Delete Specific User
+********************
+
+This endpoint exposes the Ledger API's :ref:`DeleteUserRequest <com.daml.ledger.api.v1.admin.user_management_service.DeleteUserRequest>`.
+
+HTTP Request
+============
+
+- URL: ``/v1/user/delete``
+- Method: ``POST``
+- Content-Type: ``application/json``
+- Content:
+
+.. code-block:: json
+
+    {
+      "userId": "Carol"
+    }
+
+
+Please refer to :ref:`DeleteUserRequest <com.daml.ledger.api.v1.admin.user_management_service.DeleteUserRequest>` documentation for information about the meaning of the fields.
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": true,
+      "status": 200
+    }
+
+List Users
+**********
+
+This endpoint exposes the Ledger API's :ref:`ListUsersRequest <com.daml.ledger.api.v1.admin.user_management_service.ListUsersRequest>`.
+
+HTTP Request
+============
+
+- URL: ``/v1/users``
+- Method: ``GET``
```

Comment:
Once we switch to supporting paging (will happen in the coming weeks) extra parameters need to be passed. Would you pass them via query parameters or a POST endpoint?

Reply:
I think this endpoint will not support paging. If it becomes desirable ([#6 here](https://github.com/digital-asset/daml/blob/9c03e7908bbd85ab1f8686cc581d49cc93beea92/ledger-service/http-json/README.md#development-guidelines)), it will be under a separate `POST` endpoint.

Reply:
OK.